acceptfunc.go

```diff
@@ -44,7 +44,9 @@ var defaultMsgAcceptFunc = func(dh Header) MsgAcceptAction {
 	if dh.Ancount != 0 {
 		return MsgReject
 	}
-	if dh.Nscount != 0 {
+	// IXFR request could have one SOA RR in the NS section
+	// RFC 1995, section 3: https://tools.ietf.org/html/rfc1995
```

Comment:
There's no need to include the URL. There are only two places in the library that do that, everywhere else just references the RFC number.

Reply:
OK, fixed